src/plugins/intel_gna/tests/deprecated/unit/engines/gna/I8_quantisation_test.cpp

```diff
@@ -9,8 +9,10 @@
 #include "frontend/layer_quantizer.hpp"
 #include "gna_matcher.hpp"
 #include <ie_core.hpp>
+#include <backend/gna_limitations.hpp>
```

Comment:
please use local scope includes for openvino code.

Reply:
ok